test-e2e/build/fixtures/no-deps/package.json

```diff
@@ -7,12 +7,20 @@
       [
         "bash",
         "-c",
-        "echo '#!/bin/bash\necho #{self.name}' > #{self.target_dir / self.name}"
+        "echo 'print_endline \"#{self.name}\"' > #{self.target_dir /}test.ml"
       ],
-      "chmod +x $cur__target_dir/$cur__name"
+      [
+        "ocamlopt",
+        "-o",
+        "#{self.target_dir / self.name}.exe",
+        "#{self.target_dir /}test.ml"
+      ]
     ],
     "install": [
-      "cp $cur__target_dir/$cur__name $cur__bin/$cur__name"
+      "cp $cur__target_dir/$cur__name.exe $cur__bin/$cur__name"
     ]
+  },
+  "dependencies": {
+    "ocaml": "~4.6.1"
```

Comment:
For these to work on Windows, we'll need to use this for now:
    "ocaml": "esy-ocaml/ocaml#6aacc05"

(Windows needs esy-ocaml/ocaml#3 to fix up the environment variables, but it's not published to `npm` as the ternary/conditional isn't compatible with older versions of `esy`).

Reply:
Using `esy-ocaml/ocaml#6aacc05` seems to work locally, should all tests use that for now? cc @andreypopp